### DISCUSSION TITLE
can't add reviewers to pull request

### DISCUSSION THREAD

**Discussion by anthonyllll** [community/community#50266] [Pull Requests]
### Select Topic Area

Bug

### Body

I'm a collaborator of a repo and try to add reviewers to my pull request. But unlike others who have a "Gear" button at "Reviewers" section, I cannot edit the reviewers in a PR. There's just no place to manage it. Does anybody have idea how to solve this? Appreciate it. 
![image](https://user-images.githubusercontent.com/107391757/225727316-3901b564-aaa4-4dbd-9b97-24c2f05833bb.png)

**Comment by sumitst05** [comment]
It is possible that you do not have the necessary permissions to add reviewers to a pull request in the repository. If you are a collaborator but not an owner or administrator of the repository, the repository owner may have restricted your permissions.

To check your permissions, you can go to the repository settings and look at the "Collaborators & Teams" section. Make sure that you have been granted the necessary permissions to "write" to the repository.

If you do have the necessary permissions and are still unable to add reviewers to your pull request, you may want to try the following troubleshooting steps:
- Refresh the page and try again
- Log out and log back into your GitHub account
- Try using a different web browser or clearing your browser cache
- Check if the repository has any third-party integrations that may be interfering with the "Reviewers" section

If none of these troubleshooting steps work, you may want to reach out to the repository owner or administrator for assistance.

:]

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @anthonyllll ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
